pkgs/games/tome2/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, makeDesktopItem, ncurses, libX11, boost, cmake }:
+
+let
+  pname = "tome2";
+  description = "A dungeon crawler similar to Angband, based on the works of Tolkien";
+
+  desktopItem = makeDesktopItem {
+    desktopName = pname;
+    name = pname;
+    exec = "@out@/bin/${pname}-x11";
+    icon = pname;
+    terminal = "False";
+    comment = description;
+    type = "Application";
+    categories = "Game;RolePlaying;";
+    genericName = pname;
+  };
+
+in stdenv.mkDerivation {
+  inherit pname;
+  version = "2.4";
+
+  src = fetchFromGitHub {
+    owner = "tome2";
+    repo = "tome2";
+    rev = "4e6a906c80ff07b75a6acf4ff585b47303805e46";
+    sha256 = "06bddj55y673d7bnzblk8n01z32l6k2rad3bpzr8dmw464hx4wwf";
+  };
+
+  buildInputs = [ ncurses libX11 boost ];
+
+  nativeBuildInputs = [ cmake ];
+
+  cmakeFlags = [
+    "-DSYSTEM_INSTALL=ON"
+  ];
+
+  postInstall = ''
+    mkdir -p $out/share/applications
+    cp ${desktopItem}/share/applications/*.desktop $out/share/applications
+  '';
+
+  meta = with stdenv.lib; {
+    inherit description;
+    license = licenses.unfree;
+    maintainers = with maintainers; [ cizra ];
+    platforms = with platforms; [ "x86_64-linux" ];  # Most probably works fine on other platforms, but I don't have the hardware to test.
```

Comment:
If you have no reason to believe it won't build on other platforms I'd suggest setting to `platforms.all`. When the bot builds it will confirm if this is correct or not.

Reply:
Let's try

Reply:
Let's try